myriad-scheduler/src/main/java/com/ebay/myriad/scheduler/NMProfile.java

```diff
@@ -26,14 +26,14 @@
     /**
      * Number of CPU advertised to YARN Resource Manager.
      */
-    private Long cpus;
+    private Double cpus;
 
     /**
      * Memory in MB advertised to YARN Resource Manager.
      */
-    private Long memory;
+    private Double memory;
 
```

Comment:
What does YARN advertise here?  Will this allow me to put 2.5 cpus into profiles, get 2.5 cpus from mesos, and have it truncated later?

This has issues: 
1) If we truncate later we could be rejecting perfectly good offers of 2 cpus.
2) If we actually create the task with 2.5 cpus and Yarn advertises 2 cpus, we waste cpus.

This should be investigated.  Maybe require integer values in the config, on implicitly floor them with a LOG.warn